index.js

```diff
@@ -167,11 +167,22 @@ async function bundle(items, options) {
 		or a timestamped file (for the moment) 
 		in case we're bundling many web pages.
 	 */
-	const output_path =
-		options.output ||
-		(items.length === 1
-			? `${slugify(items[0].title || 'Untitled page')}.pdf`
-			: `percollate-${Date.now()}.pdf`);
+	let output_path;
+
+	if (options.individual && options.output) {
+		if (!fs.existsSync(options.output)) {
```

Comment:
Great catch! This seems to be the case for any `--output`, regardless of whether it's called with `--individual` or not. I propose we use `fs.ensureDirSync` from the [`fs-extra`](https://github.com/jprichardson/node-fs-extra) package, for brevity:

```js
const fs = require('fs-extra');
const path = require('path');
// ...
fs.ensureDirSync(path.dirname(options.output))
```